packages/deployment/scripts/deploy-gov-v2.ts

```
@@ -0,0 +1,221 @@
+            type: 'confirm',
+            name: 'confirmed',
+            message: kleur
+              .yellow()
+              .bold(
+                '\n⚠️  WARNING: This step will transfer ownership to the timelock contract.\n' +
+                  'After this step, all changes must go through governance.\n' +
+                  'Are you sure you want to proceed?',
+              ),
+            initial: false,
+          })
+          if (confirmed) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)